packages/core/saved-objects/core-saved-objects-utils-server/src/saved_objects_test_utils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { isEqual } from 'lodash';
+
+/**
+ * Determines if a given Set is equal to another given Set. Set types must be the same, and comparable.
+ *
+ * @param setA the first Set to compare
+ * @param setB the second Set to compare
```

Comment:
__optional nit:__ (just for the consistency, here and below)
```suggestion
 * @param setA The first Set to compare
 * @param setB The second Set to compare
```